SNIP-25.md

```diff
@@ -0,0 +1,87 @@
+# SNIP-25 - Improved privacy for [SNIP-20](/SNIP-20.md) tokens
+
+This document outlines updates to the SNIP-20 specification with the introduction of `decoys`, which are designed to strengthen privacy measures by ensuring that account information remains concealed during transaction processes. This is accomplished by intermixing real accounts with 'decoy' accounts, thus making it difficult for potential attackers to discern the actual accounts being updated. Moreover, randomness is integrated into the execution messages, which offers an additional layer of protection by creating uncertainty in storage access patterns.
```

Comment:
I think a very basic explaination of what a decoy is could be helpful. We jump into "which are designed to strengthen privacy measures..." without explaining first what this is